Dwolla.Client/Rest/ResponseBuilder.cs

```diff
@@ -7,17 +7,40 @@ namespace Dwolla.Client.Rest
 {
     public interface IResponseBuilder
     {
+        Task<RestResponse> Build(HttpResponseMessage response);
         Task<RestResponse<T>> Build<T>(HttpResponseMessage response);
     }
 
     public class ResponseBuilder : IResponseBuilder
     {
+        public async Task<RestResponse> Build(HttpResponseMessage response)
+        {
+            using (var content = response.Content)
+                return await Build(content, response);
+        }
+
         public async Task<RestResponse<T>> Build<T>(HttpResponseMessage response)
         {
             using (var content = response.Content)
                 return await Build<T>(content, response);
         }
 
+        private static async Task<RestResponse> Build(HttpContent content,
+            HttpResponseMessage response)
+        {
+            if (content == null) return Error(response, "Response content is null");
```

Comment:
Please pull these strings into constants in this file since they're used in more than one place now. Something like
`private const string NullResponse = "Response content is null";`
`private const string JsonException = "Exception parsing JSON";`